frontend/src/Login.jsx

```diff
@@ -1,20 +1,37 @@
-/* eslint-disable brace-style */
-/* eslint-disable consistent-return */
-/* eslint-disable no-console */
-/* eslint-disable no-shadow */
 /* eslint-disable no-unused-vars */
-import React, { useState } from 'react';
+import React, { useContext, useEffect } from 'react';
+import { Link, Switch } from 'react-router-dom';
 import axios from 'axios';
 import Form from './Form';
+import authContext from './authContext';
 import styles from './Login.module.css';
 
 function Login() {
-  const [user, setUser] = useState([]);
+  const { setAuth } = useContext(authContext);
+
+  const linkStyle = {
+    color: 'white',
+    textDecoration: 'none',
+    fontSize: 20,
+  };
+
+  async function checkLogin() {
+    try {
+      const response = await axios.get('http://localhost:5000/login');
+      return true;
+    } catch (error) {
+      console.log(error);
+      return false;
+    }
```

Comment:
This should catch 200 and 304 for success - 304 means no one was logged in so no logout